eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -754,6 +755,67 @@ votingSpec =
       it
         "committee member can't vote on committee update when sandwiched between other votes"
         ccVoteOnConstitutionFailsWithMultipleVotes
+      it "CC cannot ratify if below quorum" $ do
+        modifyPParams $ \pp ->
+          pp
+            & ppGovActionLifetimeL .~ EpochInterval 3
+            & ppDRepVotingThresholdsL
+              .~ def
+                { dvtUpdateToConstitution = 1 %! 2
+                }
+            & ppCommitteeMinSizeL .~ 2
+            & ppCommitteeMaxTermLengthL .~ EpochInterval 10
+        (khDRep, _, _) <- setupSingleDRep 1_000_000
+        let dRepCred = KeyHashObj khDRep
+        ccColdCred0 <- KeyHashObj <$> freshKeyHash
+        ccColdCred1 <- KeyHashObj <$> freshKeyHash
+        electionGovAction <-
+          submitGovAction $
+            UpdateCommittee
+              SNothing
+              mempty
+              ( Map.fromList
+                  [ (ccColdCred0, EpochNo 10)
+                  , (ccColdCred1, EpochNo 10)
+                  ]
+              )
+              (3 %! 5)
+        submitYesVote_ (DRepVoter dRepCred) electionGovAction
+        logAcceptedRatio electionGovAction
+        passNEpochs 3
+        checkProposalsEmpty
+        ccHotKey0 <- registerCommitteeHotKey ccColdCred0
+        ccHotKey1 <- registerCommitteeHotKey ccColdCred1
+        let newConstitutionUrl = fromJust $ textToUrl 64 "newconstitution"
+        constitutionChangeId <-
+          submitGovAction $
+            NewConstitution
+              SNothing
+              Constitution
+                { constitutionScript = SNothing
+                , constitutionAnchor =
+                    Anchor
+                      newConstitutionUrl
+                      (hashAnchorData $ AnchorData "123456789")
+                }
+        submitYesVote_ (DRepVoter dRepCred) constitutionChangeId
+        resignCommitteeColdKey ccColdCred0 SNothing
+        submitYesVote_ (CommitteeVoter ccHotKey0) constitutionChangeId
+        submitYesVote_ (CommitteeVoter ccHotKey1) constitutionChangeId
+        passEpoch
+        logAcceptedRatio constitutionChangeId
+        logToExpr =<< lookupGovActionState constitutionChangeId
```

Comment:
Are these log statements necessary to keep, or where they used for debugging? Is there value to keep them?

Reply:
I'd keep them around, might help if somebody runs into problems with this test. They won't show up anyways if the test succeeds